CHANGELOG.md

```diff
@@ -181,6 +181,12 @@ The following rules are now deprecated:
 - Fix [#609](https://github.com/biomejs/biome/issues/609) `useExhaustiveDependencies`, by removing `useContext`, `useId` and `useSyncExternalStore` from the known hooks. Contributed by @msdlisper
 - Fix `useExhaustiveDependencies`, by removing `useContext`, `useId` and `useSyncExternalStore` from the known hooks. Contributed by @msdlisper
 
+#### New features
```

Comment:
The changelog line should not be here

Reply:
I think i changed it after the rebase.

But why we don't have the Unreleased section anymore ?

Reply:
We remove it before the release, otherwise it breaks the generation of the release, then the first PR to land after the release will add the unreleased section

Reply:
Ok, i'll add